boards/st/nucleo_n657x0_q/doc/index.rst

```diff
@@ -108,70 +114,134 @@ USART1. Default settings are 115200 8N1.
 Programming and Debugging
 *************************
 
-NUCLEO_N657X0_Q board includes an ST-LINK/V3 embedded debug tool interface.
-This probe allows to flash the board using various tools.
+STM32N6570_DK board includes an ST-LINK/V3 embedded debug tool interface.
```

Comment:
ok so what's the name of that board then? :) 
Overview says "Nucleo-144", the board's `full_name` is Nucleo N657x0-Q, and here you have something different? (and then ST website seems to have NUCLEO-N657X0-Q with capital X btw :) )

Reply:
Copy pasta. Reverting back to NUCLEO_N657X0_Q

Reply:
good :) please fix `full_name` too to use capital X, if you have a chance